### DISCUSSION TITLE
Android app notifications broken with LHC 4.46?

### DISCUSSION THREAD

**Discussion by PeopleInside** [LiveHelperChat/livehelperchat#2136] [General]
Hi,

I discovered that new chat are no more notified on the mobile Android app.
Also seems no notification are sent also if the chat is in progress and a new message is sent from the visitor and the app is opened in background.

Could you confirm the same issue?

Even maybe in the browser.
Is something changed in the mew chat notification and assignment to operators?
Some settings I have to change to revert back how was working in the past where I get notification on the browser and on the app on new chats?

Thanks

I uninstalled the app, reinstalled but still not working.
I expect to get a notification as in the past when a new chat is incoming... while no more notifications are sent from the Android app even if all notification settings are turned on on the Android app device settings.

Currently I'm unable to get notified about new chat on my Android device. Notification are no more working as in the past for me on the Android app., I get no notifications.

I have also disabled auto chat assignment as is no more alerting me about new chats, in the past was working. Now with that disabled on a PC desktop I have no issue to be notified on new chats but on Android mobile device still not get any notifications.

**Comment by remdex** [comment]
It can be that notifications library has expired. So just use Telegram :)

**Reply by PeopleInside** [reply]
> It can be that notifications library has expired.

What this means: you need fix or update the app?

> So just use Telegram :)

I have to see if I'm able to install.
The video tutorial is that one https://www.youtube.com/watch?v=wObbEaeopRU?

Seems not to easy ahaha but I can try.

**Reply by remdex** [reply]
If I find time I'll look. Won't happen soon :) it's not so hard to install now :) and yes watch youtube :)

**Reply by PeopleInside** [reply]
Thank you, I done successfully!
The only note I have is that when operator digit a message from Telegram user never see it his typing.
Anyway I will open a GitHub issue to track when the Android app will return to work with notifications.
Thanks for your help and great software and the nice developer and person you are.

https://github.com/LiveHelperChat/livehelperchat/issues/2137

**Reply by remdex** [reply]
I don't think possible to track from telegram is typing as telegram does not send this event :)

**Reply by shahmirf** [reply]
Hi 
I have the same issue, notifications on both IOS and android app has been disabled, which is very annoying, is there any way to solve this issue?
Thanks.

**Reply by remdex** [reply]
> Hi 
> 
> I have the same issue, notifications on both IOS and android app has been disabled, which is very annoying, is there any way to solve this issue?
> 
> Thanks.

Just use telegram

**Reply by PeopleInside** [reply]
> Hi I have the same issue, notifications on both IOS and android app has been disabled, which is very annoying, is there any way to solve this issue? Thanks.

https://youtu.be/wObbEaeopRU
https://github.com/LiveHelperChat/telegram

**Comment by shahmirf** [comment]
Thanks for the response.
We are not authorised to use Telegram, whats app on companies mobile phones, so the mobile app was the best solution and unfortunately it’s not working anymore.

**Reply by PeopleInside** [reply]
This is frustrating, I'm sorry to read that you cannot workaround the issue using the telegram app.
The only suggestion I can give at the moment, also if is not the same of having a working mobile app is to setup an email notification on the department to be alerted about new chats, if you don't have a lot of chats.
You can read more about this here: https://forum.livehelperchat.com/viewtopic.php?pid=3960

Another solution can be use Facebook messenger. I never test this but searching on the web I found this:
https://github.com/LiveHelperChat/fbmessenger
https://livehelperchat.com/integration-with-facebook-messenger-464a.html

You can maybe also consider a commercial support to have the mobile notifications issue resolved sooner. I don't know if with some bounty/donation to the developer a fix can come sooner. 
Live Helper Chat has also commercial support available https://livehelperchat.com/commercial-offers-20c.html

If you want monitor the mobile app notification issue you can follow the issue I created:
https://github.com/LiveHelperChat/livehelperchat/issues/2137
but currently there is no ETA for the fix.

I resolved with Telegram for now, for be notified on mobile.
Maybe you can be helped by the Facebook Messenger or if you are a commercial activity you can consider to ask for commercial support for resolve the mobile app notification issues.

This is my suggestions.
I'm not part of the Live Helper Chat team developer, I'm just a passionate user that are not into any business. I don't work actually.

The developer of live helper chat is a great person, kind, available and very capable.
Live helper chat is the best live chat I haver tried and seen.

I hope this reply can help you a little bit more.
Have a nice day and week.
